File: programming/labaratory-work-1/solutions/solution-4/main.c
#include <stdio.h>

int main() {
  int n = 0;
  int min_n = 0;
  int min_i = 0;

  for (int i = 1; n != 0 || i == 1; i++) {
    scanf_s("%d", &n);

    if ((n < min_n || i == 1) && n != 0) {
      min_n = n;
      min_i = i;
    }
  }

  printf_s("%d", min_i);

  return 0;
}

File: programming/labaratory-work-1/solutions/solution-4/go/main.go
package main

import "fmt"

func main() {
	var n int
	var minN int
	var minI int

	for i := 1; n != 0 || i == 1; i++ {
		fmt.Scanf("%d", &n)

		if (n < minN || i == 1) && n != 0 {
			minN = n
			minI = i
		}
	}

	fmt.Printf("%d", minI)
}